08-single-page-web-apps/mongo_mongoose/readme.md

```diff
@@ -374,19 +326,22 @@ http://docs.mongodb.org/manual/core/write-operations-introduction/
 )
 ```
 
-### You do (10 min): Update all restaurants in 20001 to be in DC.
+### You do (10 min):
+
+Update a restaurants to have a new key-value par `{state: "DC"}`
```

Comment:
I recommend you pose the problem in English and then discuss the technical solution.